scripts/__init__.py

```diff
@@ -0,0 +1,24 @@
+PYTHON_EXTENSIONS_PATHS = [
+    LOADER_DIR
+] + PYTHON_EXTENSIONS_PATHS
```

Comment:
These are _undefined names_ in Python code.
$ `flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics`
```
./scripts/__init__.py:2:5: F821 undefined name 'LOADER_DIR'
    LOADER_DIR
    ^
./scripts/__init__.py:3:5: F821 undefined name 'PYTHON_EXTENSIONS_PATHS'
] + PYTHON_EXTENSIONS_PATHS
    ^
```